examples/tracing-grpc/src/client.rs

```diff
@@ -1,10 +1,23 @@
 use hello_world::greeter_client::GreeterClient;
 use hello_world::HelloRequest;
 use opentelemetry::{global, propagation::Injector};
-use tracing::*;
-use tracing_futures::Instrument;
-use tracing_opentelemetry::OpenTelemetrySpanExt;
-use tracing_subscriber::prelude::*;
+use opentelemetry_sdk::{propagation::TraceContextPropagator, trace::TracerProvider};
+use opentelemetry_stdout::SpanExporter;
+
+use opentelemetry::{
+    trace::{SpanKind, TraceContextExt, Tracer},
+    Context, KeyValue,
+};
+
+fn init_tracer() {
+    global::set_text_map_propagator(TraceContextPropagator::new());
+    // Install stdout exporter pipeline to be able to retrieve the collected spans.
+    let provider = TracerProvider::builder()
+        .with_simple_exporter(SpanExporter::default())
```

Comment:
Since we have tokio here can we use batch span processor here?

Reply:
Good point. Updated to use the batch span processor with Tokio runtime.